modules/exploits/windows/http/manageengine_endpoint_central_saml_rce_cve_2022_47966.rb

```diff
@@ -57,15 +65,16 @@ def initialize(info = {})
               'Platform' => 'win',
               'Arch' => ARCH_CMD,
               'Type' => :windows_command,
-              'DefaultOptions' => { 'Payload' => 'cmd/windows/powershell/meterpreter/reverse_tcp' }
+              'DefaultOptions' => { 'Payload' => 'cmd/windows/powershell/meterpreter/reverse_tcp' },
```

Comment:
It looks like this payload doesn't work on my environment (ManageEngine Endpoint Central version 10.1.2228.5 on Windows Server 2019). I recommend using a [fetch payload](https://docs.metasploit.com/docs/development/developing-modules/guides/how-to-use-fetch-payloads.html#what-are-fetch-payloads) instead, which is a command-based payload that let you run any binary payloads. They are shorter and I believe size matters here.

```suggestion
              'DefaultOptions' => { 'Payload' => 'cmd/windows/https/x64/meterpreter/reverse_tcp' },
```

See my comment below about prefixing with `cmd /c`. It looks like it is needed to run any command-based payload.

Reply:
This and the prefix were tested successfully for `servicedesk_plus` and `endpoint_central`